suppresions.supp

```diff
@@ -0,0 +1,2 @@
+undefined-behavior:deps/icu-small/source/common/uloc_tag.cpp
```

Comment:
there's a typo in the filename

Reply:
also, don't we need to configure ubsan so that it uses this file?

Reply:
It's being passed as UBSAN_OPTIONS